ogr/services/gitlab/pull_request.py

```diff
@@ -103,6 +103,17 @@ def commits_url(self) -> str:
     def head_commit(self) -> str:
         return self._raw_pr.sha
 
+    @property
+    def merge_commit_sha(self) -> str:
+        return self._raw_pr.merge_commit_sha
+
+    @property
+    def merge_commit_status(self) -> CommitStatus:
+        if self._raw_pr.merge_status == "can_be_merged":
+            return CommitStatus.mergeable
+        else:
+            return CommitStatus.notmergeable
```

Comment:
Here, we can convert all possible values to the newly created `Enum`. As we discussed at the arch meeting, we want to differentiate e.g. `pending` and `notmergeable`. So, I would not treat all statuses that are not `can_be_merged` as `notmergeable` since it can't easily be a false statement.